meetings/2023-09-19.md

```diff
@@ -0,0 +1,140 @@
+
+# W3C Solid Community Group: Special Topic Meeting
+
+* Date: 2023-09-19 14:00 UTC
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Next meetings: https://github.com/solid/specification/discussions/555
+
+
+## Present
+* [Michiel de Jong](https://michielbdejong.com)
+* Reza Soltani
+* Tim Berners-Lee
+* elf Pavlik
+* Wouter Termont
+* mrkvon
```

Comment:
@mrkvon , two things:

1) can we normalise your name to "michal/mrkvon" (occurs in other minutes) - see suggestion below:

```suggestion
* michal/mrkvon
```

or perhaps stick to "mrkvon" only in the minutes (no problem to update the other minutes).

2) If you join the CG - and very much welcome to! - we'll use the same name as it appears in the CG participants list.

Reply:
I asked @mrkvon on matrix about this, and he had no problem with using either his first name or his nick. But if it's for consistency with other minutes, then we can sure use the double notation.